experimental/packages/otlp-proto-exporter-base/src/platform/browser/OTLPProtoExporterBrowserBase.ts

```diff
@@ -35,28 +37,19 @@ export abstract class OTLPProtoExporterBrowserBase<
 ExportItem,
 ServiceRequest
 > extends OTLPExporterBaseMain<ExportItem, ServiceRequest> {
-  private _send!: SendFn;
 
   constructor(config: OTLPExporterConfigBase = {}) {
     super(config);
   }
 
-  private _sendPromise(
-    objects: ExportItem[],
-    onSuccess: () => void,
-    onError: (error: OTLPExporterError) => void
-  ): void {
-    const promise = new Promise<void>((resolve, reject) => {
-      this._send(this, objects, resolve, reject);
-    })
-      .then(onSuccess, onError);
-
-    this._sendingPromises.push(promise);
-    const popPromise = () => {
-      const index = this._sendingPromises.indexOf(promise);
-      this._sendingPromises.splice(index, 1);
-    };
-    promise.then(popPromise, popPromise);
+  private getExportRequestProto(
```

Comment:
Private members need to be prefixed with `_`.

```suggestion
  private _getExportRequestProto(
```